### PR TITLE
Fixes image loads

### PR DESCRIPTION
**Pull request by 7urkm3n**
This commit fixes image load of the users (avatar, profile pictures from dm, unfollowers and search)

### CODE REVIEW

**Comment by 7urkm3n** [comment]
Feel free to give a suggestion on `css` part, I have used temporary solution with margins on the avatar.

img.style.margin = '-1px 0 0 -15px';

File: browser/js/funcs.js [hunk 1]
```diff
@@ -346,7 +346,14 @@ function quoteText (text) {
...
   const settingsButton = document.querySelector('.settings');
-  settingsButton.style.backgroundImage = `url(${url})`;
+
+  img = new Image;
+  img.crossOrigin = 'Anonymous';
+  img.src = url;
+  img.classList.add('thumb');
+  img.style.margin = '-1px 0 0 -15px';
+  settingsButton.appendChild(img);
+  // settingsButton.style.backgroundImage = `url(${url})`;
```
Comment (ifedapoolarewaju) [inline browser/js/funcs.js:356]:
```suggestion
```

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "IGdm",
-  "version": "3.0.2",
+  "version": "3.0.3",
```
Comment (ifedapoolarewaju) [inline package.json:3]:
```suggestion
  "version": "3.0.2",
```

**Review by ifedapoolarewaju — APPROVED** [review]
thank you for the fix!

**Pull request merged**